src/config/berlin/xhain.ts

```diff
@@ -192,18 +192,20 @@ const xhain: DistrictConfig = {
       ],
     },
     gastro: {
-      currentCampaign: 'xhain2021',
+      currentCampaign: 'xhain2022',
       path: 'terrassen',
+      // Allow making applications without having signed up for
+      // 'expression of interest'
       directSignup: true,
       timeline: {
         // date constructor uses 0-based month number, i.e. january is 0
```

Comment:
Btw, I had trouble with this in https://github.com/FixMyBerlin/fixmy.frontend/blob/develop/src/pages/XhainWayToSchool/index.tsx#L60 and switched to providing a date string which works a lot better IMO since it does not have this super weird off-by-one-issue. It should be possible to use that here too, IMO.